lib/slack/renderer/push.js

```diff
@@ -13,24 +13,32 @@ class Push extends Message {
     this.push = push;
   }
 
+  get commits() {
+    return this.push.commits.map((commit) => {
+      const message = commit.message.split('\n')[0];
+      const sha = commit.id.substring(0, 8);
+      return `<${commit.url}|\`${sha}\`> - ${message}`;
+    }).join('\n');
+  }
+
   getCore() {
     const noOfCommits = this.push.commits.length;
     const commits = noOfCommits === 1 ? 'commit' : 'commits';
     const branch = `\`${this.push.ref.replace('refs/heads/', '')}\``;
     const pushed = this.push.forced ? 'force-pushed' : 'pushed';
     const text = `<${this.push.compare}|${noOfCommits} new ${commits}> ${pushed} to ${branch}`;
     return {
-      fallback: text,
```

Comment:
We should definitely keep having a fallback for the push message. Otherwise it'll look really weird in mobile/desktop notifications